### DISCUSSION TITLE
Pinned Discussions Unclear

### DISCUSSION THREAD

**Discussion by kovitikus** [community/community#3061] [Discussions]
I didn't realize that the pinned discussions show up on top with these gaudy, ugly as sin backgrounds. I wasn't sure what they were when I saw them, just that it was bright colors. Didn't bother to "read" them, just kept moving down the page to the actual threads. Maybe if there were some title for this section of the Discussions that says "Pinned Discussions", that would make me understand why they are there. The colors are not only gaudy, but clearly they didn't convey what the heck they were supposed to. You could improve the recognition of what the section is for, take up far less screen space, and get rid of the clashing colors with a simple section title.

I'm also expecting to see a pin icon next to the discussion themselves in the feed.

![GitHub Pinned Discussions Feedback](https://i.imgur.com/RPUKgKU.png)

**Comment by panteliskassotis** [comment]
Valid points.

**Comment by Florian-Schoenherr** [comment]
Less radical idea:
Just add a big pin icon to the big cards and like he said, to the threads AND let the repo owner (or whoever can edit pinned discussion) decide what background to use on the big cards.
(or consider to just copy from pinned issues)

**Comment by friday** [comment]
I agree. Not that I care about the backgrounds, but I don't think this "banner" format is nearly as helpful as the plain old conventional pins that lets you put helpful messages on top of their specific sub topics.
